src/components/StoreChoises.js

```diff
@@ -1,27 +1,126 @@
 import React from 'react';
-import {Image, View, Text, StyleSheet, TouchableOpacity} from 'react-native';
+import {
+  Image,
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  Alert,
+} from 'react-native';
+import Toast from 'react-native-toast-message';
 import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import sword from '../assets/images/icon-sword.png';
+import shield from '../assets/images/icon-shield.png';
+import agility from '../assets/images/icon-agility.png';
+import coin from '../assets/images/coin.png';
+import {useAuthContext} from '../contexts/AuthContext';
+import axios from 'axios';
+
+export const StoreChoises = ({item}) => {
+  const {user, setUser} = useAuthContext();
+
+  const image = () => {
```

Comment:
You could create a separate component that receives the affected_attribute as a param and return the correct image. And use it like: `<EffectImage atb={item.affected_atribute} />`